make/docker-compose.tpl

```diff
@@ -6,16 +6,18 @@ services:
     env_file:
       - ./common/config/log/env
     restart: always
+    read_only: true
     volumes:
       - /var/log/harbor/:/var/log/docker/:z
     ports:
-      - 127.0.0.1:1514:514
+      - 127.0.0.1:1514:10514
     networks:
       - harbor
   registry:
     image: vmware/registry:2.6.2-photon
     container_name: registry
     restart: always
+    read_only: true
```

Comment:
with this restrict will it be difficult for later issue investigation?

Reply:
Thanks make sense I'll remove this flag from compose template.

However let's make sure the images are ready for `read_only`